<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageDialog</class>
 <widget class="QDialog" name="MessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>پیام</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/resources/icons/info.png</normaloff>:/icons/resources/icons/info.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>عملیات با موفقیت انجام شد.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/icons/resources/icons/info.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>B Nazanin</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>بستن</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
